collector/paths.go

```diff
@@ -44,5 +44,10 @@ func rootfsStripPrefix(path string) string {
 	if *rootfsPath == "/" {
 		return path
 	}
-	return strings.TrimPrefix(path, *rootfsPath)
+	stripped := strings.TrimPrefix(path, *rootfsPath)
+	if stripped == "" {
+		return "/"
+	} else {
```

Comment:
You can remove the `else` block here and just `return stripped`

Reply:
In fact, not! The issue this fixes is where `path` (the variable here, the mountpoint) == `rootfs.path` (the cli arg), so we would strip, for example, `"/host"` using the string `"/host"` and end up with `""`.

Reply:
see the changes made to the test for this feature in:

- `collector/fixtures_bindmount/proc/mounts`
- `collector/filesystem_linux_test.go`

Reply:
No, what @pgier is saying is this is equivalent:

```go
if stripped == "" {
	return "/"
}
return stripped
```

Reply:
Right, sorry it wasn't that clear in my original comment.  go-lint in vscode complains when I try to have an else block at the end of a function that just does a return.

Reply:
Ahh, fair enough! I've just modified and pushed up! Thanks @SuperQ and @pgier !